docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1975 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        T_subseq_isfinite = core.rolling_isfinite(T, m)
</pre><p>We don't need this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>